### PR TITLE
ci: combine sqllogic test handlers to reduce jobs

### PR DESCRIPTION
**Pull request by everpcpc**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Closes #issue

### CODE REVIEW

File: .github/mergify.yml [hunk 1]
```diff
@@ -19,11 +19,11 @@ queue_rules:
       - check-success=test_stateful_standalone_linux
       - check-success=test_stateful_cluster_linux
       # sqllogic tests
-      - check-success=test_sqllogic_http_base_standalone_linux
-      - check-success=test_sqllogic_mysql_ydb_standalone_linux
-      - check-success=test_sqllogic_clickhouse_query_standalone_linux
-      - check-success=test_sqllogic_mysql_base_cluster_linux
-      - check-success=test_sqllogic_http_query_cluster_linux
+      - check-success=test_sqllogic_base_standalone_linux
```
Comment (xudong963) [inline .github/mergify.yml:22]: I think we also need to check: standalone, crdb, duckdb.

File: .github/mergify.yml [hunk 2]
```diff
@@ -19,11 +19,11 @@ queue_rules:
...
+      - check-success=test_sqllogic_ydb_standalone_linux
+      - check-success=test_sqllogic_query_standalone_linux
+      - check-success=test_sqllogic_base_cluster_linux
```
Comment (xudong963) [inline .github/mergify.yml:25]: ditto

**Review by Xuanwo — APPROVED** [review]

**Review by xudong963 — APPROVED** [review]

**Review by PsiACE — APPROVED** [review]

**Comment by everpcpc** [comment]
```
Fri, 06 Jan 2023 04:49:03 GMT 16_0001_flashback:                                                      [ OK ]
Fri, 06 Jan 2023 04:59:03 GMT 20_0005_mysql_auth:                                                     [ FAIL ] - Timeout!
```

`20_0005_mysql_auth` stuck for 10 min in standalone stateless test

**Comment by BohuTANG** [comment]
> ```
> Fri, 06 Jan 2023 04:49:03 GMT 16_0001_flashback:                                                      [ OK ]
> Fri, 06 Jan 2023 04:59:03 GMT 20_0005_mysql_auth:                                                     [ FAIL ] - Timeout!
> ```
> 
> `20_0005_mysql_auth` stuck for 10 min in standalone stateless test

Likely OK now? Most tests are less than 10m.

**Pull request merged**